runtime/src/apis.rs

```diff
@@ -579,3 +621,73 @@ impl_runtime_apis! {
 		}
 	}
 }
+
+fn decode_runtime_event_v1(event: &super::RuntimeEvent) -> Option<Vec<u8>> {
+	use super::*;
+	use codec::Encode;
+
+	match event {
+		RuntimeEvent::Sudo(e) => match e {
```

Comment:
So, filtering is possible only on this subset of events? Dont we want to add bridge events to this set?